appendix.rst

```diff
@@ -202,7 +202,7 @@ below with their respective functionality.
 ``N``
 =====
 
-#. **SINGULARITY_NAME**: Specifies a custom image name.
+#. **apptainer_NAME**: Specifies a custom image name.
```

Comment:
```suggestion
#. **SINGULARITY_NAME**: Specifies a custom image name.
```
There are several more below that im not going to individually mark, use this PR diff to locate and fix them please